beetmoverscript/templates/fennec_nightly.yml

```diff
@@ -128,5 +128,9 @@ mapping:
       destinations:
         - latest-{{ branch }}-{{ platform }}/{{ locale_prefix }}robocop.apk
         - {{ upload_date }}-{{ branch }}-{{ platform }}/{{ locale_prefix }}robocop.apk
-
+    buildhub.json:
+      s3_key: latest-{{ branch }}-{{ platform }}/{{ locale_prefix }}fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json
```

Comment:
`s3_key` is solely used in checksums generation. Whatever we end up placing here, will be landing in the SHA512SUMS files (e.g. [here](https://archive.mozilla.org/pub/firefox/candidates/61.0b14-candidates/build1/SHA256SUMS)). So in this case, it's better if we simply keep the `fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json` part without its leading `latest ...` prefix.